src/libs/OptionsListUtils.js

```diff
@@ -1215,6 +1215,7 @@ function getOptions(
         };
         userToInvite = createOption([optimisticAccountID], personalDetailsExtended, null, reportActions, {
             showChatPreviewLine,
+            shouldNotFallbackToHidden: true,
```

Comment:
```suggestion
            shouldFallbackToHidden: true,
```

Right? That's what it seems to be in the function above.

Reply:
Because it should fall back to hidden, the exception to that rule implemented somewhere else.

Reply:
I don't see `shouldNotFallbackToHidden` used anywhere else in the codebase. I only see `shouldFallbackToHidden`. So what will passing `shouldNotFallbackToHidden` here do?